homeassistant/components/sonos/media_player.py

```diff
@@ -720,7 +718,8 @@ def update_media_radio(self, variables, track_info):
             ) and (
                 self.state != STATE_PLAYING
                 or self.soco.music_source_from_uri(self._media_title) == MUSIC_SRC_RADIO
-                or self._media_title in self._uri
+                and self._uri is not None
+                and self._media_title in self._uri  # type: ignore[operator]
```

Comment:
This looks odd, why did you change `or` to `and`?

(And `self._uri` is never `None` but I assume that's related to the typing which I don't know much about)

Reply:
Good catch.  That doesn't look right.  Thanks for spotting it.  Fixed in https://github.com/home-assistant/core/pull/48900